src/Credentials/ServiceAccountCredentials.php

```diff
@@ -37,12 +37,11 @@
  *   use Google\Auth\Middleware\AuthTokenMiddleware;
  *   use GuzzleHttp\Client;
  *   use GuzzleHttp\HandlerStack;
- *   use GuzzleHttp\Psr7;
  *
- *   $stream = Psr7\stream_for(file_get_contents(<my_key_file>));
  *   $sa = new ServiceAccountCredentials(
  *       'https://www.googleapis.com/auth/taskqueue',
- *       $stream
+ *       null,
+ *       '/path/to/your/json/key_file.json'
```

Comment:
Good catch. I think this is better. Although we'd be better off combining argument 2 and 3 of `ServiceAccountCredentials` like we do [here](https://github.com/google/google-api-php-client/blob/master/src/Google/Client.php#L816)